src/core/Tabs/Tabs.tsx

```diff
@@ -244,7 +248,11 @@ export const Tabs = (props: TabsProps) => {
   };
 
   return (
-    <div className={cx('iui-tabs-wrapper', `iui-${orientation}`)}>
+    <div
+      className={cx('iui-tabs-wrapper', `iui-${orientation}`, className)}
+      style={style}
+      id={id}
```

Comment:
Why not just `{...rest}`?

Reply:
Hm, then all `data-*` and `aria-*` attributes are passed not on the parent element.

Reply:
Yeah... this wrapper should have been a separate component

Reply:
Should it? Isn't it here because of selection animation?

Reply:
which one makes more sense?

```tsx
<Tabs {...billionProps} />
```
or

```tsx
<TabsWrapper>
  <Tabs />
  <TabPanel />
</TabsWrapper>
```

Reply:
When you write like this 😁 But it still would be `<Tabs {...sameProps} />`

Reply:
We would be able to get rid of these props.
![image](https://user-images.githubusercontent.com/9084735/140763242-880ead84-98fc-4b9e-b35a-bbb3c6cca2ff.png)

And we would gain extra customization on all of those parts. Right now you can't do anything with the panel content other than apply class name.